examples/step-63/jacobi.prm

```diff
@@ -0,0 +1,14 @@
+# Finite Element degree
+set Fe degree = 1
+
+# Smoother Type: SOR|Jacobi|block SOR|block Jacobi
+set Smoother type = Jacobi
+
+# DoF renumbering: no|downstream|upstream|random
+set DoF renumbering = downstream
+
+# With streamline diffusion: true|false
+set With streamline diffusion = true
+
+# Output: true|false
+set Output = false
```

Comment:
Oh, I missed that this file (and the other parameter files) doesn't have a newline at the end. Can you please address this as part of a follow-up patch, for example when you write the documentation?

Reply:
Yes.